framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -396,9 +396,10 @@ public long getToValueFromIntRange(AnnotatedTypeMirror atm) {
      * Performs pre-processing on annotations written by users, replacing illegal annotations by
      * legal ones.
      */
-    private class ValueTypeAnnotator extends TypeAnnotator {
+    protected class ValueTypeAnnotator extends TypeAnnotator {
```

Comment:
Are the changes in this file related to your bug fix?
If not, I suggest removing them from this pull request and putting them in a different pull request of their own.